pymongo/uri_parser.py

```diff
@@ -138,7 +166,7 @@ def _parse_options(opts, delim):
             if key.lower() == 'authmechanismproperties':
                 val = value
             else:
-                val = unquote_plus(value)
+                val = unquote(value)
```

Comment:
Again these new unquote_plus->unquote changes are breaking. Please revert them and mention it in the new ticket.

Reply:
Done.